jmunixusers/cs-vm-build
# Bring back Robots (poorly)

Okay so this does a couple of terrible things:
1. Reverts #378 and tries to resolve some revert conflicts introduced by #422
2. Tries to bring up support for ROS 2 Galactic (which is totally untested because I don't have access to an AMD64 host at the moment)
3. Add support for "Experimental" courses to the VM config tool. These are "hidden" behind a checkbox in the Settings dialog

This should not be used by anyone reasonable or anyone who needs their VM to work for their class(es). If it is to be tested, it should absolutely be done in a separate VM.

Steps to test:
1. Boot up a fresh VM
2. Open the config tool, go to File > Settings and change the branch and repo to:
   * **Branch**: `bring-back-robots`
   * **Repo**: `https://github.com/kylelaker/cs-vm-build`
3. Click OK and then Run
4. Close the VM Config Tool
5. Open the VM Config Tool again
6. Go back to File > Settings and enable the "Experimental" flag
7. Check the box for "CS 354"
8. Click Run
9. Report any failures (and there will likely be a lot of them)
